external-adapter/src/config.ts

```diff
@@ -5,26 +5,30 @@ import { logger } from './logger'
 const ENV_API_KEY = 'API_KEY'
 const ENV_API_ENDPOINT = 'API_ENDPOINT'
 const ENV_API_TIMEOUT = 'API_TIMEOUT'
+const ENV_API_VERBOSE_RESPONSE = 'API_VERBOSE_RESPONSE'
 const DEFAULT_API_TIMEOUT = 30000
 
 export const constants = {
   ENV_API_KEY,
   ENV_API_ENDPOINT,
   ENV_API_TIMEOUT,
   DEFAULT_API_TIMEOUT,
+  ENV_API_VERBOSE_RESPONSE,
 }
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 const cloneNoSecrets = (config: Config): Config => (({ apiKey, ...o }) => o)(config)
 
 export function getDefaultConfig(prefix = ''): Config {
   const apiKey = util.getEnv(ENV_API_KEY, prefix)
+  const timeout = util.getEnv(ENV_API_TIMEOUT, prefix)
   return {
     apiKey,
+    returnVerboseResponse: !!util.getEnv(ENV_API_VERBOSE_RESPONSE, prefix),
```

Comment:
What is this used for?

Reply:
This is used for if the adapter should return the whole data response or not. With composite adapters these responses could end up massive. Maybe we should default to true as that is the behaviour of every other current adapter?

Reply:
But this specific flag `returnVerboseResponse`, I see it being defined, but not used anywhere?

Reply:
Ahhh you are right, hadn't changed the variable name in the ea-factories/balance code to match this name.